plugins/tds61.js

```diff
@@ -644,6 +644,9 @@ tds61 = {
         // Drop the FCSUBTYPE since we don't use it
         if (attrs.FCSUBTYPE) delete attrs.FCSUBTYPE;
 
+        // Drop the the "Not Found" F_CODE. This is from the UI
+        if (attrs.F_CODE == 'Not found') delete attrs.F_CODE;
+
```

Comment:
In the first comment I was describing the current behavior and then trying to understand what the new expected behavior would be with this PR.

As for F_CODE Not found, if that is only TDS to OSM and not OSM to TDS then my fears were unfounded.